package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fluxible",
-  "version": "0.2.2",
+  "version": "0.2.3",
```

Comment:
Ah, yes. @joemcbride can you remove this change? We prefer to do versioning separately in so that the tags and commits match up correctly.